plugin/quick_scope.vim

```diff
@@ -73,7 +73,7 @@ vnoremap <silent> <plug>(QuickScopeToggle) :<c-u>call <sid>toggle()<cr>
 " Colors ---------------------------------------------------------------------
 " Detect if the running instance of Vim acts as a GUI or terminal.
 function! s:get_term()
-  if has('gui_running') || (has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR)
+  if has('gui_running') || (has('nvim') && $NVIM_TUI_ENABLE_TRUE_COLOR) || has('termguicolors')
```

Comment:
should this be `has('termguicolors') && &termguicolors`?

Reply:
Yes, it should. `$NVIM_TUI_ENABLE_TRUE_COLOR` has been deprecated and removed since [2016/05/11](https://github.com/neovim/neovim/wiki/Following-HEAD#20160511). The line should thus be:

    if has('gui_running') || has('termguicolors') && &termguicolors